plugin-build/src/main/kotlin/io/sentry/android/gradle/autoinstall/AbstractInstallStrategy.kt

```diff
@@ -68,4 +84,22 @@ abstract class AbstractInstallStrategy : ComponentMetadataRule {
             }
         }
     }
+
+    private fun parseVersion(version: String): SemVer? {
+        val semVer = parseVersionSafely(version)
+        if (semVer == null && version.endsWith(".RELEASE")) {
```

Comment:
hm, should we just rather do `parseVersionSafely(version.removeSuffix(".RELEASE")` to avoid double-parsing altogether?

Reply:
Thinking about this again it seems the right thing. Will change it.